microsoft-365/compliance/gdpr-arc-Dynamics365.md

```diff
@@ -88,7 +88,7 @@ For more GDPR-related documentation, visit https://aka.ms/gdprgetstarted.
 | ***Information Security Policies (6.2)*** | The customer should augment any existing information security policies to include protection of personal data, including policies necessary for compliance with any applicable legislation. | Microsoft policies for information security and any specific measures for the protection of personal information.<br>- Dynamics 365-ISO27001 Statement of Applicability [[5](gdpr-arc-Dynamics365.md#5)]<br>- Dynamics 365-ISO27018 Statement of Applicability [[6](gdpr-arc-Dynamics365.md#6)]<br>- SOC 2 Type 2 Audit Report [[8](gdpr-arc-Dynamics365.md#8)]<br>- 6.2 Information security policies in Compliance Manager [[15](gdpr-arc-Dynamics365.md#15)] | 24(2) |
-| ***Organization of Information Security Customer consideration (6.3)*** | The customer should, within their organization, define responsibilities for security and protection of personal data. This may include establishing specific roles to oversee privacy-related matters, including a DPO. Appropriate training and management support should be provided to support these roles. | An overview of the role of Microsoft's Data Protection Officer, the nature of his duties, reporting structure and contact information.<br>- Microsoft DPO Information [[16](gdpr-arc-Dynamics365.md#16)]<br>- Dynamics 365-ISO27001 Statement of Applicability [[5](gdpr-arc-Dynamics365.md#5)]<br>- Dynamics 365-ISO27018 Statement of Applicability [[6](gdpr-arc-Dynamics365.md#6)]<br>- 6.3 Organization of information security in Compliance Manager [[15](gdpr-arc-Dynamics365.md#15)] | (37)(1)(a), (37)(1)(b), (37)(1)(c), (37)(2), (37)(3), (37)(4), (37)(5), (37)(6), (37)(7), (38)(1), (38)(2), (38)(3), (38)(4), (38)(5), (38)(6), (39)(1)(a), (39)(1)(b), (39)(1)(c), (39)(1)(d), (39)(1)(e), (39)(2) |
+| ***Organization of Information Security Customer consideration (6.3)*** | The customer should, within their organization, define responsibilities for security and protection of personal data. This may include establishing specific roles to oversee privacy-related matters, including a DPO. Appropriate training and management support should be provided to support these roles. | An overview of the role of Microsoft's Data Protection Officer, the nature of their duties, reporting structure and contact information.<br>- Microsoft DPO Information [[16](gdpr-arc-Dynamics365.md#16)]<br>- Dynamics 365-ISO27001 Statement of Applicability [[5](gdpr-arc-Dynamics365.md#5)]<br>- Dynamics 365-ISO27018 Statement of Applicability [[6](gdpr-arc-Dynamics365.md#6)]<br>- 6.3 Organization of information security in Compliance Manager [[15](gdpr-arc-Dynamics365.md#15)] | (37)(1)(a), (37)(1)(b), (37)(1)(c), (37)(2), (37)(3), (37)(4), (37)(5), (37)(6), (37)(7), (38)(1), (38)(2), (38)(3), (38)(4), (38)(5), (38)(6), (39)(1)(a), (39)(1)(b), (39)(1)(c), (39)(1)(d), (39)(1)(e), (39)(2) |
```

Comment:
```suggestion
| ***Organization of Information Security Customer consideration (6.3)*** | The customer should, within their organization, define responsibilities for security and protection of personal data. This may include establishing specific roles to oversee privacy-related matters, including a DPO. Appropriate training and management support should be provided to support these roles. | An overview of the role of Microsoft's Data Protection Officer, the nature of their duties, reporting structure, and contact information.<br>- Microsoft DPO Information [[16](gdpr-arc-Dynamics365.md#16)]<br>- Dynamics 365-ISO27001 Statement of Applicability [[5](gdpr-arc-Dynamics365.md#5)]<br>- Dynamics 365-ISO27018 Statement of Applicability [[6](gdpr-arc-Dynamics365.md#6)]<br>- 6.3 Organization of information security in Compliance Manager [[15](gdpr-arc-Dynamics365.md#15)] | (37)(1)(a), (37)(1)(b), (37)(1)(c), (37)(2), (37)(3), (37)(4), (37)(5), (37)(6), (37)(7), (38)(1), (38)(2), (38)(3), (38)(4), (38)(5), (38)(6), (39)(1)(a), (39)(1)(b), (39)(1)(c), (39)(1)(d), (39)(1)(e), (39)(2) |
```